src/less/forms.less

```diff
@@ -800,6 +801,97 @@ input[type='checkbox']:focus {
   }
 }
 
+// add other attr to input to identify it in css file, similar to checked
+
+.rhinoslidingradio__input {
+  &:checked + .rhinoslidingradio__label::after {
+    transform-origin: center center;
+    transform: scale(1.5, 1.5);
+    transition: all 0.25s @global-trans-timing-cubic;
+  }
+  &:checked + .rhinoslidingradio__label.u-text-danger {
+    &::after {
+      border: 2px solid @global-color-danger;
+    }
+    &::before {
+      transform: scale(1, 1);
+      background-color: @global-color-danger;
+      transition: all 0.25s @global-trans-timing-cubic;
+    }
+  }
+  &:checked + .rhinoslidingradio__label.u-text-warning {
+    &::after {
+      border: 2px solid @global-color-warning;
+    }
+    &::before {
+      transform: scale(1, 1);
+      background-color: @global-color-warning;
+      transition: all 0.25s @global-trans-timing-cubic;
+    }
+  }
+  &:checked + .rhinoslidingradio__label.u-text-secondary {
+    &::after {
+      border: 2px solid @global-color-secondary;
+    }
+    &::before {
```

Comment:
I feel like this code could be condensed a bit if we place the shared attributes outside of the color class bracket. 

Shared attributesfor `&:checked &:before` I see as: `transform` `transition`. That way we're not writing those in three different places, and just changing the `&:after` selector which changes color with each attached color class. 

Let me know if that makes sense.